qemu/tests/cfg/block_hotplug_negative.cfg

```diff
@@ -14,16 +14,16 @@
     remove_image_stg0 = yes
     force_create_image_stg0 = yes
     kill_vm_on_error = yes
-    blk_num = 1
     blk_extra_params_stg0 = "invalid_param=on"
 
     variants:
         - @default:
         - data_plane:
             no Host_RHEL.m6
+            only virtio_blk, virtio_scsi
             iothreads = iothread0
             virtio_blk:
-                blk_extra_params_image1 = "iothread=${iothreads}"
+                blk_extra_params = "iothread=${iothreads}"
             virtio_scsi:
```

Comment:
It will occurred error:" (status: 1,    output: "qemu-kvm: -device ide-cd,id=cd1,drive=drive_cd1,bootindex=1,iothread=iothread0,bus=ide.0,unit=0: Property '.iothread' not found")"
it will still add "iothread" to ide device.